crates/cast/src/main.rs

```diff
@@ -30,14 +37,18 @@ struct Cli {
     #[clap(short = 'u', long = "url")]
     rpc_url: Option<String>,
 
-    /// Account name to be used for contract declaration; overrides account from Scarb.toml
+    /// Account name to be used for contract declaration; overrides account from Scarb.toml; should be a path if used with --keystore
```

Comment:
Let's expand the description a bit more more
```suggestion
    /// Account to be used for contract declaration; 
    /// When using keystore (`--keystore`), this should be a path to account file
    /// When using accounts file, this should be an account name
```

Something like this?